src/sage/rings/function_field/element_rational.pyx

```diff
@@ -74,10 +74,11 @@ cdef class FunctionFieldElement_rational(FunctionFieldElement):
             sage: K.<t> = FunctionField(GF(7))
             sage: t.element()
             t
-            sage: type(t.element())                                                     # needs sage.rings.finite_rings
+            sage: type(t.element())                                                     # needs sage.libs.ntl
             <... 'sage.rings.fraction_field_FpT.FpTElement'>
```

Comment:
I wonder why `ntl` is involved. Here.

Reply:
fraction_field_FpT is implemented using NTL

Reply:
OK.